test.py

```diff
@@ -1,81 +1,53 @@
+import json
 import sys
 from time import time
 from time import sleep
-import argparse
```

Comment:
It looks like the `test.py` you were working from was pretty dated. It doesn't contain argument parsing, so users cannot overrwrite the config file via an argument.